server/tests/lib/subject_page_config_content_test.py

```diff
@@ -108,6 +108,8 @@ def test_required_fields(self):
         "sustainability": [libutil.get_disaster_sustainability_config()],
     })
     for id, configs in all_configs.items():
+      if id == 'sdg':
```

Comment:
curious why you're skipping sdg config because it should also be verified in the same way?

Reply:
I think that will be a bigger change because right now topic pages for the same topic will have different charts for different places. So maybe we should revisit how we match places to configs. 

Maybe for now, we just ignore the place verification for sdg (which I see you've added), but can you add a TODO to revisit places?